app/src/ui/diff/index.tsx

```diff
@@ -650,15 +650,33 @@ export class Diff extends React.Component<IDiffProps, {}> {
     )
   }
 
-  private onCopy = (selection: string, event: Event) => {
+  private onCopy = (editor: CodeMirror.Editor, event: Event) => {
     event.preventDefault()
 
-    // Remove all the diff markers.
-    const textWithoutMarkers = selection
-      .split('\n')
-      .map(l => l.substr(1))
-      .join('\n')
-    clipboard.writeText(textWithoutMarkers)
+    // Remove the diff line markers from the copied text. The beginning of the
+    // selection might start within a line, in which case we don't have to trim
+    // the diff type marker. But for selections that span multiple lines, we'll
+    // trim it.
+    const doc = editor.getDoc()
+    const lines: ReadonlyArray<string> = (doc as any).getSelections()
```

Comment:
PR opened adding this: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/19128